pages/instructorQuestionManualGrading/instructorQuestionManualGrading.ejs

```diff
@@ -22,6 +22,32 @@
     </div>
     <div id="content" class="container">
       <div class="row">
+
+        <% if (typeof submission != null) { %>
+          <% if (submission.score == null && submission.manual_grading_user != null) { %>
+          <div class="col-lg-12 col-sm-12">
+            <div class="card mb-4">
+              <div class="card-header bg-warning text-blue">
+                <% if (authn_user.uid === user.uid) { %>
```

Comment:
What is `user` here? Is this supposed to be `grading_user`?

Reply:
This logic was updated. If authenticated user is not the manual grading user declared on the particular instance_question row, then the warning message appears.